src/lib/query/collectionOld.ts

```diff
@@ -0,0 +1,222 @@
+import { gql } from "graphql-request";
+
+export const getCollectionsQueryOld = gql`
+  query getCollectionsQuery($offset: Int!, $pageSize: Int!, $search: String) {
+    collections(
+      limit: $pageSize
+      offset: $offset
+      where: {
+        _or: [
+          { name: { _iregex: $search } }
+          { vm_address: { vm_address: { _eq: $search } } }
+        ]
+      }
```

Comment:
we are not using expression on the old query? if those minitias don't have too much data, then i'm fine with this since this is temporary anyway

Reply:
I'll fix this.